stdlib/src/builtin/simd.mojo

```diff
@@ -717,6 +718,26 @@ struct SIMD[type: DType, size: Int](
                 self.value, rhs.value
             )
 
+    # TODO there may be a better way to do this:
+    # the [__:None=None] parameter is to give this overload lower precedence,
+    # while still conforming to EqualityComparable
+    @always_inline("nodebug")
+    fn __eq__[uwu: None = None](self, rhs: Self) -> Bool:
+        """Compares two SIMD vectors using all-equal-to comparison.
+
+        This overload allows EqualityComparable conformance.
+
+        Parameters:
+            uwu: OwO.
```

Comment:
Can we give a better identifier than uwu? :)

Reply:
*sigh* I suppose.